csv_util.cc

```diff
@@ -1421,7 +1420,7 @@ xcsv_data_read(void)
      * pre-read the file to know how many data lines we should be seeing,
      * we take this cheap shot at the data and cross our fingers.
      */
-    foreach(const QString& ogp, xcsv_file.epilogue) {
+    for(const auto& ogp : qAsConst(xcsv_file.epilogue)) {
```

Comment:
In general, we're going to have a lot of noise here because Qt's foreach predates C11's range for and C17's const foreach iterators that semi-automatically create a thread pool and parallelize.